recipes/gst-plugins-good/all/conanfile.py

```diff
@@ -0,0 +1,205 @@
+            meson.build()
+
+    def _fix_library_names(self, path):
+        # regression in 1.16
+        if self.settings.compiler == "Visual Studio":
+            with tools.chdir(path):
+                for filename_old in glob.glob("*.a"):
+                    filename_new = filename_old[3:-2] + ".lib"
+                    self.output.info("rename %s into %s" % (filename_old, filename_new))
+                    shutil.move(filename_old, filename_new)
+
+    def package(self):
+        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        with tools.environment_append(VisualStudioBuildEnvironment(self).vars) if self._is_msvc else tools.no_op():
+            meson = self._configure_meson()
+            meson.install()
+
+        self._fix_library_names(os.path.join(self.package_folder, "lib"))
+        self._fix_library_names(os.path.join(self.package_folder, "lib", "gstreamer-1.0"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "gstreamer-1.0", "pkgconfig"))
+        tools.remove_files_by_mask(self.package_folder, "*.pdb")
+
+    def package_info(self):
+
+        plugins = ["alpha", "alphacolor",
+                   "apetag",
+                   "audiofx",
+                   "audioparsers",
+                   "auparse",
+                   "autodetect",
+                   "avi",
+                   "cutter",
+                   "debug",
+                   "deinterlace",
+                   "dtmf",
+                   "effectv",
+                   "equalizer",
+                   "flv",
+                   "flxdec",
+                   "goom",
+                   "goom2k1",
+                   "icydemux",
+                   "id3demux",
+                   "imagefreeze",
+                   "interleave",
+                   "isomp4",
+                   "alaw", "mulaw",
+                   "level",
+                   "matroska",
+                   "monoscope",
+                   "multifile",
+                   "multipart",
+                   "replaygain",
+                   "rtp",
+                   "rtpmanager",
+                   "rtsp",
+                   "shapewipe",
+                   "smpte",
+                   "spectrum",
+                   "udp",
+                   "videobox",
+                   "videocrop",
+                   "videofilter",
+                   "videomixer",
+                   "wavenc",
+                   "wavparse",
+                   "y4menc"]
+
+        gst_plugin_path = os.path.join(self.package_folder, "lib", "gstreamer-1.0")
+        if self.options.shared:
+            self.output.info("Appending GST_PLUGIN_PATH env var : %s" % gst_plugin_path)
+            self.cpp_info.bindirs.append(gst_plugin_path)
+            self.env_info.GST_PLUGIN_PATH.append(gst_plugin_path)
```

Comment:
```suggestion
            self.cpp_info.bindirs.append(gst_plugin_path)
            self.runenv_info.GST_PLUGIN_PATH.append(gst_plugin_path)
            self.env_info.GST_PLUGIN_PATH.append(gst_plugin_path)
```

to support `VirtualRunEnv`